.github/workflows/main.yml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# This is a basic workflow to help you get started with Actions
+
+name: WeeklyOpenApiDocsDownload
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: 
+      - 'weeklyOpenApiDocsDownload.*'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  downloadOpenApiDocs:
+    # The type of runner that the job will run on
```

Comment:
I suggest you add a skip ci option.
https://github.com/microsoftgraph/msgraph-beta-sdk-dotnet/blob/master/.github/workflows/create-pull-request.yml#L24

Reply:
This is particularly useful if you need to update the PR with a commit, so as not to trigger this GH action again.